docs/docs/admin/robots2policy.mdx

```diff
@@ -2,6 +2,7 @@
 title: robots2policy CLI Tool
 sidebar_position: 50
 ---
+> "LET'S MAKE ROBOTS.TXT GREAT AGAIN!" - [Jason Cameron](https://jsn.cam/)
```

Comment:
When was `robots.txt` great, and in what way has it declined in greatness since the publishing of the spec in 1994?